src/tuning/tuningusb.cpp

```diff
@@ -74,6 +75,10 @@ usb_tunable::usb_tunable(const char *path, const char *name) : tunable("", 0.9,
 	else if (strlen(vendor))
 		snprintf(desc, sizeof(desc), _("Autosuspend for USB device %s [%s]"), vendor, name);
 
+	str1 = product;
+	transform(str1.begin(), str1.end(), str1.begin(), ::tolower);
+	safe = str1.find("mouse") == string::npos && str1.find("keyboard") == string::npos;
```

Comment:
Should something be logged, that these are “unsafe”, and therefore ignored?

Reply:
Certainly, because these are not always unsafe to suspend, such as with machines running as (headless) home servers.